package.json

```diff
@@ -15,5 +15,9 @@
   },
   "scripts": {
     "start": "webpack-dev-server --hot"
+  },
+  "dependencies": {
+    "inferno-compat": "^3.7.1",
```

Comment:
Lock these and put them into devDependencies.
Locking means removing the `^` from the version.

Remeber to remove the yarn.lock and generate a new one once they are locked.